source/common/microkernel_architecture/microkernel_modules/instructions/AInstruction.hpp

```diff
@@ -0,0 +1,121 @@
+#ifndef AINSTRUCTION_HPP_
+# define AINSTRUCTION_HPP_
+
+# include "AInstructionBusClient.hpp"
+# include "struct_packed.h"
+# include <QByteArray>
+
+typedef unsigned int magicNumber_t;
+
+class AInstruction
+{
+protected:
+    //
+    // `struct instructionHeader_t` represents how the data is
```

Comment:
Nope, these comments explain how an instruction is structured in term of bytes
And there is nothing better to explain that, than the structures used themselves
Like for the public, private, protected, attributes, I think that this is better for the code understanding